### PR TITLE
Add files from the DFP 'iframe buster kit'

### PR DESCRIPTION
**Pull request by dlh01**
Originally downloaded from https://support.google.com/adxseller/answer/1074250?hl=en. 

Each of these files comes from the kit except `adx_iframe.html`, which comes from https://support.google.com/adxseller/answer/2696380.

I ignored some files in the kit because they appeared to me to fail the "Parameters that accept any domain" test. In some other files, I added `encodeURIComponent()` calls where they seemed necessary.

### CODE REVIEW

File: templates/admotion/afa-iframe.htm [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  if(pos==-1)continue;
+  var argname=pairs[i].substring(0,pos),value=pairs[i].substring(pos+1);
+  args[argname]=unescape(value)
+ }
+ return args
+};
+var args=ADM_getArgs();
+for(var prop in args)
+	self[prop]=args[prop];
+document.writeln('<scr' + 'ipt type="text/javascr' + 'ipt" src="' + (encodeURIComponent(args.iq) || 'http://ia.nspmotion.com/delivery/') + '?p=' + encodeURIComponent(args.p) + '&sc=' + encodeURIComponent(args.sc) + '&r=' + (Math.round(Math.random() * 9999)) + (encodeURIComponent(args.test) ? '&test=true' : '') + '"></scr' + 'ipt>');
```
Comment (mjangda) [inline templates/admotion/afa-iframe.htm]: Looks like this loads any script file passed in via the `iq` param? Don't think the `encode` protects us much.

Comment (dlh01) [inline templates/admotion/afa-iframe.htm]: Ah, yeah. Do you want me to remove this file?

Comment (mjangda) [inline templates/admotion/afa-iframe.htm]: Probably for the best.

Comment (dlh01) [inline templates/admotion/afa-iframe.htm]: Removed.

File: templates/contobox/iframebuster.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <script type="text/javascript">
+        var params = {};
+        (window.onpopstate = function () {
+            var match,
+                pl     = /\+/g,
+                search = /([^&=]+)=?([^&]*)/g,
+                decode = function (s) { return decodeURIComponent(s.replace(pl, " ")); },
+                query  = window.location.search.substring(1);
+            params = {};
+            while (match = search.exec(query)) params[decode(match[1])] = decode(match[2]);
```
Comment (philipjohn) [inline templates/contobox/iframebuster.html]: Could this `exec()` be used to execute code passed via a GET param?

Comment (nickdaugherty) [inline templates/contobox/iframebuster.html]: The `exec()` here just executes the regex (`search`) against the input `query`.

Comment (philipjohn) [inline templates/contobox/iframebuster.html]: :+1: awesome

**Pull request merged**